Adapter_oM/OrderedHashSet.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BH.oM.Adapter
+{
+    public class OrderedHashSet<T> : KeyedCollection<T, T>
```

Comment:
Do not see this used anywhere. Is this a leftover from testing?

Reply:
Now removed.